### PR TITLE
Add ember detector

### PR DESCRIPTION
**Pull request by locks**
**- Summary**

This detects that a project is using ember-cli and runs the appropriate command.

**- Test plan**

Running `netlify dev` inside an ember-cli project correctly identifies and starts the project.

**- Description for the changelog**

`netlify dev` now detects ember-cli projects.

### CODE REVIEW

File: src/detectors/ember.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const { hasRequiredDeps, hasRequiredFiles, getYarnOrNPMCommand, scanScripts } = require('./utils/jsdetect')
+
+module.exports = function() {
+  // REQUIRED FILES
+  if (!hasRequiredFiles(['package.json'])) return false
+  if (!hasRequiredFiles(['ember-cli-build.js'])) return false
+  // REQUIRED DEPS
+  if (!hasRequiredDeps(['ember-cli'])) return false
+
+  /** everything below now assumes that we are within vue */
```
Comment (lifeart) [inline src/detectors/ember.js]: ember?

Comment (locks) [reply src/detectors/ember.js]: Busted 😁 I copied the vue file over.

Comment (locks) [reply src/detectors/ember.js]: Addressed, thanks!

**Comment by RaeesBhatti** [comment]
Looks good to me. Thanks for the PR @locks

**Pull request merged**